content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -264,10 +264,13 @@ running in a second cluster. Before you begin:
     {{< /tip >}}
 
     If the global services have actual VIPs, you can use those, but otherwise we suggest
-    using IPs from the multicast range `224.0.0.0/4` that are not already allocated.
-    These IPs are not loopback addresses and are non-routable outside of a pod.
+    using IPs from the class E addresses range `240.0.0.0/4` that are preserved.
+    These IPs are not loopback addresses and are non-routable outside of a pod.    
     Application traffic for these IPs will be captured by the sidecar and routed to the
     appropriate remote service.
+    
+    **Note** Multicast addresses(224.0.0.0 ~ 239.255.255.255) should not be used as there is no route to them by default.
```

Comment:
```suggestion
{{< warning >}}
Use IPs that are not loopback addresses but also not routable outside of a pod.
Do not use multicast addresses (224.0.0.0 ~ 239.255.255.255) because there is no route to them by default.
{{< /warning >}}
```